CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## v 0.5.0 / 2017-11-14
+
+NOTE: This release breaks backward compatibility. `statsd_exporter` now uses
+a YAML configuration file. You must convert your mappings configuration to
+the new format when you upgrade.
+
+* [CHANGE] Use YAML for configuration file
+* [BUGFIX] Fix matching empty statsd metric components #105
```

Comment:
I would even consider this a feature. This wasn't _intended_ to work before, but we allow it now because there was a demonstrated need.